pkgs/tools/misc/pastebinit/default.nix

```diff
@@ -1,51 +1,55 @@
 { lib
 , stdenv
-, fetchurl
-, fetchpatch
+, fetchFromGitHub
 , python3
+, libxslt
+, docbook_xsl
+, installShellFiles
+, callPackage
 }:
 stdenv.mkDerivation rec {
-  version = "1.5";
+  version = "1.6.2";
   pname = "pastebinit";
 
-  src = fetchurl {
-    url = "https://launchpad.net/pastebinit/trunk/${version}/+download/${pname}-${version}.tar.bz2";
-    sha256 = "0mw48fgm9lyh9d3pw997fccmglzsjccf2y347gxjas74wx6aira2";
+  src = fetchFromGitHub {
+    owner = pname;
+    repo = pname;
+    rev = version;
+    hash = "sha256-vuAWkHlQM6QTWarThpSbY0qrxzej0GvLU0jT2JOS/qc=";
   };
 
-  buildInputs = [
-    (python3.withPackages (p: [ p.distro ]))
+  patches = [
+    ./use-drv-etc.patch
   ];
 
-  patchFlags = [ "-p0" ];
+  nativeBuildInputs = [
+    libxslt
+    installShellFiles
+  ];
 
-  patches = [
-    # Required to allow pastebinit 1.5 to run on Python 3.8
-    ./use-distro-module.patch
-    # Required to remove the deprecation warning of FancyURLopener
-    ./use-urllib-request.patch
-    # Required because pastebin.com now redirects http requests to https
-    (fetchpatch {
-      name = "pastebin-com-https.patch";
-      url = "https://bazaar.launchpad.net/~arnouten/pastebinit/pastebin-com-https/diff/264?context=3";
-      sha256 = "0hxhhfcai0mll8qfyhdl3slmbf34ynb759b648x63274m9nd2kji";
-    })
+  buildInputs = [
+    (python3.withPackages (p: [ p.distro ]))
   ];
 
+  buildPhase = ''
+    xsltproc --nonet ${docbook_xsl}/xml/xsl/docbook/manpages/docbook.xsl pastebinit.xml
+  '';
+
   installPhase = ''
     mkdir -p $out/bin
     mkdir -p $out/etc
     cp -a pastebinit $out/bin
+    cp -a utils/* $out/bin
     cp -a pastebin.d $out/etc
-    substituteInPlace $out/bin/pastebinit --replace "'/etc/pastebin.d" "'$out/etc/pastebin.d"
+    substituteInPlace $out/bin/pastebinit --subst-var-by "etc" "$out/etc"
+    installManPage pastebinit.1
   '';
 
   meta = with lib; {
-    homepage = "https://launchpad.net/pastebinit";
+    homepage = "https://stgraber.org/category/pastebinit/";
```

Comment:
hmm, the manpage still lists https://launchpad.net/pastebinit as homepage - I guess either are as good as the other.